Keas.Mvc/Services/IdentityService.cs

```diff
@@ -260,8 +260,23 @@ public async Task<string> BulkLoadPeople(string ppsCode, string teamslug)
                         }
                     }
                     else
-                    {
-                        warning.Append($" IAM ID {id.IamId} failed to save.");
+                    {
+                        //Try to get name for Iam Id 
+                        var extraName = string.Empty;
+                        try
+                        {
+                            var iamInfoResults = await clientws.People.Get(id.IamId);//  //.Kerberos.Search(KerberosSearchField.iamId, id.IamId);
+                            if (iamInfoResults.ResponseData.Results.Length > 0)
+                            {
+                                extraName = $"({iamInfoResults.ResponseData.Results.First().OFullName}) ";
+                            }
+                        }
+                        catch (Exception e)
+                        {
```

Comment:
If you aren't catching you shouldn't declare the exception var.  So like `catch {}` only